pycbc/io/live.py

```diff
@@ -426,9 +419,9 @@ def upload(self, fname, gracedb_server=None, testing=True,
                     displayName=['SNR timeseries']
                 )
                 gracedb.writeLog(
-                    gid, 'PSD plot upload',
-                    filename=psd_series_plot_fname,
-                    tag_name=['psd'], displayName=['PSDs']
+                    gid, 'ASD plot upload',
+                    filename=asd_series_plot_fname,
+                    tag_name=['asd'], displayName=['ASDs']
```

Comment:
I suggest keeping the `tag_name` to `psd`. I am not sure how GraceDB would react to the change.